src/staff_webapp/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# choose node
+FROM node:20-alpine AS build_angular
+
+# set working directory inside the container
+WORKDIR /staff_webapp
+
+# copy package.json & package-lock.json
+COPY package*.json ./
+
+# install dependencies/packages
+RUN npm install -g @angular/cli
+RUN npm install
+
+# copy all the project files
+COPY . .
+
+# react default expose port
+EXPOSE 4200
+
+# build our webapp
+CMD ["ng", "serve", "--host", "0.0.0.0"]
```

Comment:
ng is a development focused service. Eventually, we will want an actual deployment service to handle the distribution of our webapp. 
It looks like you referenced [This](https://dev.to/rensjaspers/how-to-containerize-an-angular-app-for-production-20ba) site to build your Dockerfile. It recommends Caddy 2, which seems like a pretty solid option if we don't want to build it ourselves.

This topic has been added to our meeting agenda for our next meeting, so don't worry about it now

Reply:
Yes, it is currently in development environment. We'll remove `EXPOSE 4200` and `CMD ["ng", "serve", "--host", "0.0.0.0"]` and do `RUN ng build` or `RUN npm build` without installing @angular/CLI to finish building the webapp (stage 1). 
I don't think what we had on this Dockerfile right now indicating we're using Caddy 2. In stage 2, we can set up the production environment (stage 2) with either `FROM nginx:latest` or `FROM caddy:latest`

Reply:
I see, that makes sense. In which case feel free to leave it in or remove it, whichever you'd prefer 👍